### DISCUSSION TITLE
Default form options

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5584] [Feature Requests]
**Describe the bug**
Users should have a choice between several form display options:

- White background, white enabled fields, gray disabled fields (System default)
- Off-white background, off-white enabled fields, gray disabled fields
- Dark background, dark enabled fields, black disabled fields

**Comment by grantfitzsimmons** [comment]
Started to be fixed on this branch https://github.com/specify/specify7/issues/1232

**Comment by grantfitzsimmons** [comment]
Will start on another branch and bring into agent merging

**Comment by CarolineDenis** [comment]
@grantfitzsimmons do you mean add new pref to choose from this 3 options?

**Comment by maxpatiiuk** [comment]
hmm, I am struggling to come up with how this would be represented in the preferences.
would this be a separate theme picker from the main theme picker,
or would the main theme picker get a third option (off-white)?

maybe it would be best to just add a checkbox like "Accent form fields" or "Contrast form fields" or etc - if checked, fields are light gray in light mode and dark gray in dark mode - if not checked, fields match background color - in either case, disabled fields will be the opposite

**Comment by grantfitzsimmons** [comment]
<img width="453" alt="image" src="https://github.com/specify/specify7/assets/37256050/3db095c8-aac3-4f4e-b567-8763fbd10490">

**Comment by grantfitzsimmons** [comment]
My dream
